packages/gatsby-theme-docs/src/components/content-pagination.js

```diff
@@ -10,6 +10,8 @@ import Card from './card';
 import Spacings from './spacings';
 import TextSmall from './text-small';
 
+const trimTrailingSlash = url => url.replace(/(\/?)$/, '');
```

Comment:
It seems to be rather an artifact in this PR but  my experience is that it's a bad idea to change gatsby's default behavior concerning the URL structure.   Gatsby is assuming  and generating trailing slash URLs in many situations so I think it's not ideal to try to work against its defaults.   
Using "real" folders  is also more interoperable with various hosting options because the real output of the static builds are folders with index.html files in them. 

Opinionated topic, I know..

Reply:
This is just to properly match the URL (slug) with the one defined manually in e.g. the `navigation.yaml`. Yes, Gatsby adds the trailing slash but the navigation links are maintained manually, which is error prone.